diff --git a/dist/LabelerServer.js b/dist/LabelerServer.js
index 5e57c5912c17427740c987c8cc72030452588094..85b4de87c82174a84527341fc1788e79d56aee4d 100644
--- a/dist/LabelerServer.js
+++ b/dist/LabelerServer.js
@@ -1,8 +1,8 @@
 import "@atcute/ozone/lexicons";
 import { XRPCError } from "@atcute/client";
-import { fastifyWebsocket } from "@fastify/websocket";
 import { createClient } from "@libsql/client";
-import fastify from "fastify";
+import websocket from "@fastify/websocket";
+import Fastify from "fastify";
 import { parsePrivateKey, verifyJwt } from "./util/crypto.js";
 import { formatLabel, labelIsSigned, signLabel } from "./util/labels.js";
 import { excludeNullish, frameToBytes } from "./util/util.js";
@@ -55,15 +55,24 @@ export class LabelerServer {
             this.db = createClient({ url: "file:" + (options.dbPath ?? "labels.db") });
         }
         this.dbInitLock = this.initializeDatabase();
-        this.app = fastify();
-        void this.app.register(fastifyWebsocket).then(() => {
+        this.app = Fastify();
+
+        (async () => {
+            await this.app.register(websocket);
+
+            // WebSocket route MUST be registered before wildcard
+            this.app.get(
+                "/xrpc/com.atproto.label.subscribeLabels",
+                { websocket: true },
+                this.subscribeLabelsHandler,
+            );
+
             this.app.get("/xrpc/com.atproto.label.queryLabels", this.queryLabelsHandler);
             this.app.post("/xrpc/tools.ozone.moderation.emitEvent", this.emitEventHandler);
-            this.app.get("/xrpc/com.atproto.label.subscribeLabels", { websocket: true }, this.subscribeLabelsHandler);
             this.app.get("/xrpc/_health", this.healthHandler);
             this.app.get("/xrpc/*", this.unknownMethodHandler);
             this.app.setErrorHandler(this.errorHandler);
-        });
+        })();
     }
     /**
      * Initializes the database with the required schema.
@@ -92,7 +101,13 @@ export class LabelerServer {
     }
     start(portOrOptions, callback = () => { }) {
         if (typeof portOrOptions === "number") {
-            this.app.listen({ port: portOrOptions }, callback);
+            this.app.listen(
+                {
+                    port: portOrOptions,
+                    host: "0.0.0.0",
+                },
+                callback,
+            );
         }
         else {
             this.app.listen(portOrOptions, callback);
